src/panels/config/integrations/integration-panels/dynalite/dynalite-config-panel.ts

```diff
@@ -0,0 +1,279 @@
+import "@material/mwc-button";
+import "@polymer/app-layout/app-header/app-header";
+import "@polymer/app-layout/app-toolbar/app-toolbar";
+import "@polymer/paper-input/paper-input";
+import "@polymer/paper-listbox/paper-listbox";
+import "@polymer/paper-item/paper-item";
+import {
+  css,
+  CSSResultArray,
+  customElement,
+  html,
+  LitElement,
+  property,
+  internalProperty,
+  TemplateResult,
+} from "lit-element";
+import "../../../../../components/ha-card";
+import "../../../../../components/ha-menu-button";
+import "../../../../../layouts/ha-app-layout";
+import "../../../../../components/ha-settings-row";
+import "../../../../../components/ha-paper-dropdown-menu";
+import "../../../../../components/ha-switch";
+import { haStyle } from "../../../../../resources/styles";
+import { HomeAssistant } from "../../../../../types";
+import { getConfigEntries } from "../../../../../data/config_entries";
+
+@customElement("dynalite-config-panel")
+class HaPanelDevDynalite extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property() public narrow!: boolean;
+
+  private entryData;
+
+  private _activeOptions = ["on", "init", "off"];
+
+  @internalProperty() private _name = "";
+
+  @internalProperty() private _host = "";
+
+  @internalProperty() private _port = "";
+
+  @internalProperty() private _fade = "";
+
+  @internalProperty() private _active = "";
+
+  @internalProperty() private _auto_discover = "";
+
+  @internalProperty() private _poll_timer = "";
+
+  protected render(): TemplateResult {
+    return html`
+      <ha-app-layout>
+        <app-header slot="header" fixed>
+          <app-toolbar>
+            <ha-menu-button
+              .hass=${this.hass}
+              .narrow=${this.narrow}
+            ></ha-menu-button>
+            <div main-title>${this.localStr("description_settings")}</div>
+          </app-toolbar>
+        </app-header>
+
+        <div class="content">
+          <ha-card header="${this.localStr("description_system")}">
+            <div class="card-content">
+			  <ha-settings-row .narrow=${this.narrow}>
+				<span slot="heading">${this.localStr("name")}</span>
+				<span slot="description">${this.localStr("name_long")}</span>
+                <paper-input
+                  class="flex"
+                  .label=${this.localStr("name")}
+                  name="name"
+                  type="string"
+                  .value=${this._name}
+                  @value-changed=${this._handleChange}
+                >
+			  </ha-settings-row>
```

Comment:
Are you using prettier?

Reply:
isn't it running automatically with commit? it changes my code. 
this is my first frontend PR so probably many newbie errors. apologies. any specific way to run prettier?

Reply:
IF you are using VS code and you have it installed and you have it set up as your formatter and you have it set up to format on save then you are running prettier.

But Use VS Code, Download the recommended Plugins in the .vscode folder
Set prettier as your formatter, you can look that up on google. and then in VS code change the setting for format on save to be checked

Reply:
ok. will give it a try. wasn't useing vs code